x/auth/ante/ante.go

```diff
@@ -4,30 +4,43 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth/signing"
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
+	feegrantkeeper "github.com/cosmos/cosmos-sdk/x/feegrant/keeper"
 )
 
 // NewAnteHandler returns an AnteHandler that checks and increments sequence
 // numbers, checks signatures & account numbers, and deducts fees from the first
 // signer.
 func NewAnteHandler(
-	ak AccountKeeper, bankKeeper types.BankKeeper,
+	ak AccountKeeper, bankKeeper types.BankKeeper, feeGrantKeeper *feegrantkeeper.Keeper,
```

Comment:
Would be great to not just add more and more positional parameters. Instead I suggest an `Options` struct.